---
title: "Tahiri Lab - Research"
layout: textlay
excerpt: "Tahiri Lab -- Research"
sitemap: false
permalink: /software/
---

## Data and software
*Developing new tools is often necessary to address new questions. Our team develops statistical and mathematical approaches, implements them and releases them as open-source software to ensure reproducibility and reliability of analyses.*

We will develop interactive software to implement our new algorithms and make them available on the web. The newly developed software will be accompanied by a user guide as well as few examples of technical applications, including the one covered in the second section. The software will significantly contribute to the discovery of groups of gene trees having similar intragroup and intergroup evolutionary histories.

We propose to develop user-friendly software for visualizing the classification of phylogenetic trees that will become an essential tool for evolutionary and molecular biologists.

All algorithms will be implemented and released as free and open-source software to make analyses reproducible and reliable. We plan to implement the programs for the Macintosh, UNIX, and Windows platforms.

<img src="{{ site.url }}{{ site.baseurl }}/images/pubpic/DoubleRecViz_2.png" class="img-responsive" width="100%" style="float: center" />
<div style="text-align: right">
  Source : Kuitche, E; Qi, Y; Tahiri, N; Parmer, J; Ouangraoua, A. (2020) <br>
</div>


### Principal Investigator:

Nadia Tahiri (University of Sherbrooke)


### Related publication:

Kuitche, E; Qi, Y; Tahiri, N; Parmer, J; Ouangraoua, A. (2020). DoubleRecViz: A Web-Based Tool for Visualizing Transcript-Gene-Species reconciliation. Bioinformatics

### FUNDING:

Programme MITACS Accelerate

